components/clp-py-utils/clp_py_utils/decorators.py

```diff
@@ -0,0 +1,20 @@
+import functools
+
+
+def exception_default_value(default):
+    """
+    A function decorator that returns the value specified by `default` when an uncaught exception
+    occurs in the decorated function. Otherwise, the decorated function's return value is returned.
```

Comment:
```suggestion
    A function decorator that returns the value specified by `default` when an uncaught exception
    occurs in the decorated function. Otherwise, the decorated function's return value is returned.
    :param default: The value to return upon catching an uncaught exception.
    :return: The decorator.
```

Sorry, missed this in the previous suggestion.